images/user-data.yaml

```diff
@@ -6,23 +6,18 @@ stages:
          # It will partition a device including the given filesystem label or part label (filesystem label matches first)
          device:
            label: COS_RECOVERY
-         # Only last partition can be expanded
-         # expand_partition:
-         #   size: 4096
          add_partitions:
            - fsLabel: COS_STATE
-             size: 8192
+             # 15GB for COS_STATE, the disk should be at least of 16GB
```

Comment:
will this be enough? on my tests the recovery partition was 2Gb so this should be at least 17Gb?

Reply:
Yes you are right, gonna write it should be bigger than 17Gb